example-model/src/main/java/com/appunite/rx/example/model/model/AddPost.java

```diff
@@ -0,0 +1,26 @@
+package com.appunite.rx.example.model.model;
+
+import javax.annotation.Nonnull;
+
+public class AddPost {
```

Comment:
probably this post has a lot of comment with normal post so you can use inheritance